components/partial/MDXStyleWrapper.tsx

```diff
@@ -87,6 +87,12 @@ const MDXStyleWrapper = ({ children }: { children: any }) => (
       > img {
         ${imageStyles.styles}
       }
+
+      span {
```

Comment:
Adding `<InlineCode>` styles to code blocks within `<span>`. <span> us used within Collapsible components in the FAQ pages.